### ISSUE TITLE
Suggestion to reverse two of the last changes

### ISSUE THREAD

**Issue by ArmorKeik** [CriticalFlaw/TF2HUD.Fixes#20]
Hi, it's me again.

I was really excited to hear that you updated this mod today.
Thank you very much for keeping it in support.

But besides adding and correcting very important things, I noticed that you changed the chat color and took out a background from the player table.

I honestly think the default color was much better.
And also that the table had more detail with the background.

![unknown-1-01.jpeg](https://user-images.githubusercontent.com/87909542/135698599-a3e73d3c-14c5-4f5c-a78a-f5699f03f2a4.jpeg)

![vi-1-01.jpeg](https://user-images.githubusercontent.com/87909542/135698604-021b3c1d-e302-465b-b757-4063316aff46.jpeg)

So I suggest adding an option for those who want the table and chat as it was before, so they can choose to their liking users.

**Comment by CriticalFlaw** [comment]
Hello. The change to the chat color was not submitted to the workshop and can be reverted by deleting `resoucres/chatscheme.res` if you so wish.

The background panel was removed because it would always be displayed in tournament spectator mode and this was the only reliable way of fixing that.

**Comment by ArmorKeik** [comment]
Oh, I see.

Well, thanks for telling me how to manually change the chat color (I think you should add that info as a readme.txt or somewhere else).

And isn't there a way to return the player table to how it was before. By deleting some file or modifying another one?

**Comment by CriticalFlaw** [comment]
The only change to the player table is that the background panel was removed when in match mode. It can be restored by going to: https://github.com/CriticalFlaw/TF2HUD.Fixes/blob/master/resource/ui/hudmatchstatus.res#L221 and adding
```
if_match
{
   "visible" "1"
}
```

But that's up to the individual user to restore if they care enough to do so. The background won't scale if you were to decide to make the player table larger.

The chat change could be added or the change could be reverted if enough people have an issue with that change. It was a requested change after all.

**Comment by ArmorKeik** [comment]
Oh, I see.

Well, thanks for getting back to me.

The only bug I see now is that, when lowering the player counter in Arena mode, it overlaps with the boss health bar in the "Unofficial FF2" game mode.

This only happens when alive, if you are dead the health bar goes down (I guess to avoid joining the grey lines in spectator mode, although I disable those lines with a command in autoexec).

![Captura de pantalla (55)](https://user-images.githubusercontent.com/87909542/135763928-62e0b287-450a-4cba-bba4-4e11210ac2a9.png)

![Captura de pantalla (58)](https://user-images.githubusercontent.com/87909542/135763932-dbcfe287-af81-4cfb-8691-86b15f531462.png)


Although I think this is not something you can fix, as it happens in a community game mode only, so I think you should report it to its creator.

Translated with www.DeepL.com/Translator (free version)

**Comment by CriticalFlaw** [comment]
That's correct. The intent with this project is to address issues found in officially supported gamemodes.
